src/server/server.ts

```diff
@@ -104,6 +114,7 @@ registerRoute('get', '/run/:host/:id', (req, res) => {
                 returnUrl: '',
                 origin: currentConfig.editorUrl,
                 host: host,
+                assets: assets,
```

Comment:
Inlined the function, like you suggested, so there's no need for the extra variable declaration.